data/hai/hai ships.txt

```diff
@@ -805,6 +805,9 @@ ship "Pond Strider" "Pond Strider (Railgun)"
 
 
 
+# This default build is intended for the player,
+# whom the Unfettered do not want to arm properly for fighting the Hai (it lacks energy gen)
+# but instead with general plunder and utility in mind.
```

Comment:
```suggestion
```

Stock/shipyard builds are supposed to be general functional builds, not optimized or specialized for a particular thing. As such, I don't think we need a comment explaining why this isn't an optimized build.

Reply:
yeah you're right, I just wanted to make sure nobody would use the default unfet variant against Hai and be like "its baaaad"
but that much should be obvious

Reply:
lol. If it's bad, and a regular opponent such that they have had experience, time, and opportunity to customize a loadout for that opponent (and the Unfettered clearly have against the Hai), then they should have their own configuration in a variant. Which already exists. :)